### DISCUSSION TITLE
How do I make new nodes use FORMAT="markdownPatternFormat"?

### DISCUSSION THREAD

**Discussion by reagle** [freeplane/freeplane#1698] [Help]
I like markdown and appreciate the Freeplane supports it. Currently, though, to use it throughout I have to go into MM files and add `FORMAT="markdownPatternFormat"` to nodes. And even after that, new nodes I create are not in markdown mode. Is there a way to make a whole map markdown, including new nodes?

**Comment by macmarrum** [comment]
You can go to `Format->Manage Styles->Edit styles`, select the Style **Default** and change **Core text -> Format** to `Markdown`
You can also change **Content types -> For details/note** to `Markdown`

This will affect all nodes in your mind map, because all nodes' style is derived from **Default**

https://docs.freeplane.org/user-documentation/styles.html#style-editing

**Comment by EdoFro** [comment]
Hi @reagle,
1. go to menu `Format->Manage Styles->Edit styles (Ctrl+F11)`
2. in the styles map select the style you want to change. In your case, if tou want all nodes of your map to use Markdown style, then select the Default Style
3. select Core Text / Format : Markdown
4. Content Types For Details and Note: Markdown

![image](https://github.com/freeplane/freeplane/assets/35700575/13aa2700-edff-486b-be1f-926b89d09eb6)
